etc/kayobe/ansible/build-ofed.yml

```diff
@@ -0,0 +1,103 @@
+---
+- name: Build OFED packages
+  become: true
+  hosts: ofed-builder
+  gather_facts: false
+  tasks:
+    - name: Extend the home logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_home
+        size: +2G
+        resizefs: true
+
+    - name: Extend the temporary logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_var_tmp
+        size: +2G
+        resizefs: true
+
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - kpartx
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-modules-extra
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+          - createrepo
+          - cmake-filesystem
+          - libnl3-devel
+          - python3-devel
+        state: latest
+        update_cache: true
+
+    - name: Set GRUB default index to saved
```

Comment:
Could we use our [grub command line](https://github.com/stackhpc/ansible-collection-linux/tree/main/roles/grubcmdline) collection for this?

Reply:
I think that role is for updating the kernel command line, but here we're setting the default kernel entry for GRUB to boot.